build_all_wheels.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# build compiled wheel versions of pike with multiple different python versions
+
+# ensure system build requirements are met (see README.md)
+
+# any additional arguments supplied will be passed to the final
+# `setup.py bdist_wheel` command and may be used to upload the
+# packages to artifactory
+#  ./build_all_wheels.sh upload -r http://artifactory01.prod.sea1.west.isilon.com/artifactory/api/pypi/pypi-local
+
+# recommend the use of `pyenv` to install and activate
+# multiple python versions before running the script.
+
+set -euo pipefail
+
+# XXX: py3.9 not supported by internal artifactory
+versions=(python2 python3.6 python3.7 python3.8)
+
+for v in ${versions[@]}
+do
+    rm -rf .build/$v && \
+    virtualenv -p $v .build/$v && \
+    .build/$v/bin/pip install 'build' && \
+    .build/$v/bin/python -m build && \
+    .build/$v/bin/python setup.py bdist_wheel --universal $*
```

Comment:
i don't think it makes sense to build a universal wheel when we have compiled extensions.

https://packaging.python.org/guides/distributing-packages-using-setuptools/#wheels

> If your project also supports Python 2 and contains no C extensions, then you should create what’s called a Universal Wheel ...
>
> Only use this setting if your project does not have any C extensions and supports Python 2 and 3.

Reply:
ok

Reply:
Nice catch... TIL